app/utils/encrypt.js

```diff
@@ -0,0 +1,10 @@
+import CryptoJS from 'crypto-js';
```

Comment:
You can also use `bcrypto` for this which is already a dependency of hsd and all the algorithms are written in both C and JS

Reply:
replaced `crypto-js` with `bcrypto`